src/index.ts

```diff
@@ -231,6 +233,70 @@ function launchMetals(
       );
     }
 
+    registerCommand("metals.new-scala-file", async () => {
+      const currentDoc = await workspace.document;
+      const currentPath = currentDoc.uri;
+      const parentDir = path.dirname(currentPath);
```

Comment:
So currently we aren't giving the user a lot of option about where this is being created. It will just grab the current directory they are in and stick it there. I went back and forth on what would be best with this. One option would be to show the entire path, and allow them to just edit it, but form a UX perspective that seems a bit odd. Plus I think it can be a bit confusing. Do I just put the path? Do I end with a slash? Is there a prompt for the directory, and then also the name? It just gets dirty really quickly. Thoughts?

Reply:
I think it's fine to default to the current directory and see how the users feel about this. If it becomes an issue, we can easily add more options.